app/src/main/java/com/w3engineers/unicef/util/helper/InAppShareUtil.java

```diff
@@ -0,0 +1,239 @@
+package com.w3engineers.unicef.util.helper;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.ResolveInfo;
+import android.graphics.Bitmap;
+import android.net.wifi.WifiConfiguration;
+import android.net.wifi.WifiInfo;
+import android.net.wifi.WifiManager;
+import android.os.Environment;
+import android.support.annotation.Nullable;
+
+import com.google.zxing.BarcodeFormat;
+import com.google.zxing.MultiFormatWriter;
+import com.google.zxing.common.BitMatrix;
+import com.w3engineers.ext.strom.App;
+import com.w3engineers.unicef.telemesh.R;
+import com.w3engineers.unicef.telemesh.ui.inappshare.InstantServer;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.lang.reflect.Method;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * ============================================================================
+ * Copyright (C) 2019 W3 Engineers Ltd. - All Rights Reserved.
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ * Created by: Mimo Saha on [13-Mar-2019 at 6:33 PM].
+ * Email:
```

Comment:
please remove the unused comments from here.

Reply:
Thanks for sharing but this xml file is not exist in current codebase.